src/components/ParserComponent.tsx

```diff
@@ -299,11 +311,12 @@ export default class ParserComponent extends React.Component<ParserInterface,
       try {
         if (this.props.fileType.mimeName === '.csv' +
             ',text/csv' && this.isValid(temp)) {
+          console.log('Target file is a .csv');
           await this.parseCsv(fileEvent);
         }
       } catch (e) {
         alert('Wrong file type was uploaded.');
-        console.log('Wrong file was uploaded.');
+        console.error('Wrong file was uploaded.');
```

Comment:
consider changing this to e.toString(), so that you print the error that's caught